src/pages/landing/weekend-trading/components/_hero.js

```diff
@@ -10,8 +10,32 @@ import { LinkButton } from 'components/form'
 import device from 'themes/device.js'
 
 const BackgroundWrapper = styled(Background)`
-    background-color: transparent;
-    height: 100%;
+    background-size: cover;
+    background-position: bottom right;
+    @media ${device.desktopL} {
+        height: 70rem !important;
+    }
+    @media ${device.desktop} {
+        height: 65rem !important;
+    }
+    @media ${device.tabletL} {
+        height: 155rem !important;
+    }
+    @media ${device.tablet} {
+        height: 136rem !important;
+    }
+    @media ${device.tabletS} {
+        height: 105rem !important;
+    }
+    @media ${device.mobileL} {
+        height: 95rem !important;
+    }
+    @media ${device.mobileM} {
+        height: 90rem !important;
+    }
+    @media ${device.mobileS} {
+        height: 80rem !important;
+    }
```

Comment:
Can we not set the height based on the `vw` instead of setting all these media queries?

Reply:
to have best view of background image used this way.

Reply:
Can we remove the `!important` here?

Reply:
done